blark/iec.lark

```diff
@@ -759,6 +759,7 @@ case_list: case_list_element ( "," case_list_element )*
 
 ?case_list_element: subrange
                   | integer_literal
+                  | bit_string_literal
```

Comment:
Needs an annotation update for `CaseMatch` to include `BitString` https://github.com/engineerjoe440/blark/blob/fe1d37d11c5d455a36d8ef8819b084550f952e7a/blark/transform.py#L2549

Reply:
Well... what do you know... look at that...
![image](https://user-images.githubusercontent.com/33275230/168329525-8f4b64f8-65f2-4939-8344-bdfeebea0355.png)

Reply:
Totally valid syntax... :confused: :+1:

Reply:
Should be all set, now! Thank you for pointing this out!